src/components/svgs/stations/qingdao-metro-int.tsx

```diff
@@ -0,0 +1,196 @@
+            return [0, names[0].split('\\').length * LineHeight[oY] + 12];
+        } else if (oX === 'right' && oY === 'bottom') {
+            return [7, names[0].split('\\').length * LineHeight[oY] + 7];
+        } else if (oX === 'left' && oY === 'middle') {
+            return [-13, 0];
+        } else if (oX === 'right' && oY === 'middle') {
+            return [13, 0];
+        } else return [0, 0];
+    };
+
+    const [textX, textY] = getTextOffset(nameOffsetX, nameOffsetY);
+    const textAnchor = nameOffsetX === 'left' ? 'end' : nameOffsetX === 'right' ? 'start' : 'middle';
+
+    return (
+        <g id={id} transform={`translate(${x}, ${y})`}>
+            <circle
+                id={`stn_core_${id}`}
+                r={8}
+                stroke="black"
+                strokeWidth="0.67"
+                fill="white"
+                onPointerDown={onPointerDown}
+                onPointerMove={onPointerMove}
+                onPointerUp={onPointerUp}
+                style={{ cursor: 'move' }}
+            />
+            <g transform={`translate(${textX}, ${textY})`} textAnchor={textAnchor}>
+                <MultilineText
+                    text={names[0].split('\\')}
+                    fontSize={10}
+                    lineHeight={10}
+                    grow="up"
+                    baseOffset={1}
+                    className="rmp-name__zh"
+                />
+                <MultilineText
+                    text={names[1].split('\\')}
+                    fontSize={5.5}
+                    lineHeight={5.5}
+                    grow="down"
+                    baseOffset={1.5}
+                    className="rmp-name__en"
+                />
+            </g>
+        </g>
+    );
+};
+
+/**
+ * qingdaoMetroIntStation specific props.
+ */
+export interface QingdaoMetroIntStationAttributes extends StationAttributes {
+    nameOffsetX: NameOffsetX;
+    nameOffsetY: NameOffsetY;
+}
+
+const defaultQingdaoMetroIntStationAttributes: QingdaoMetroIntStationAttributes = {
+    ...defaultStationAttributes,
+    nameOffsetX: 'right',
+    nameOffsetY: 'top',
+};
+
+const qingdaoMetroIntAttrsComponent = (props: AttrsProps<QingdaoMetroIntStationAttributes>) => {
+    const { id, attrs, handleAttrsUpdate } = props;
+    const { t } = useTranslation();
+
+    const fields: RmgFieldsField[] = [
+        {
+            type: 'textarea',
+            label: t('panel.details.stations.common.nameZh'),
+            value: attrs.names[0].replaceAll('\\', '\n') ?? defaultQingdaoMetroIntStationAttributes.names[0],
+            onChange: val => {
+                attrs.names[0] = val.toString().replaceAll('\n', '\\');
+                handleAttrsUpdate(id, attrs);
+            },
+            minW: 'full',
+        },
+        {
+            type: 'textarea',
+            label: t('panel.details.stations.common.nameEn'),
+            value: attrs.names[1].replaceAll('\\', '\n') ?? defaultQingdaoMetroIntStationAttributes.names[1],
```

Comment:
No need for `.replaceAll('\\', '\n')` any more. It's only used for backwards compatibility for older stations that use `\\` as a line breaker.